src/floor/SimulationVars.java

```diff
@@ -5,8 +5,9 @@
 public  class SimulationVars {
 	public static int numberOfElevators = 3;
 	public static int numberOfFloors = 3;
-
-	public static int elevatorTravelTime = 500;
+	
+	public static int timeScalar = 10;
```

Comment:
kinda a future nitpick thing but we should probably make gets and setters for these vars